src/frontend/pages/game.tsx

```diff
@@ -32,6 +36,12 @@ export default function Home(): ReactElement {
     setTimeLeft(10);
   }
 
+  const handleTake = (event) => {
+    event.preventDefault();
```

Comment:
what exactly is the default we're preventing here?

Reply:
I'm honestly not too familiar with prevent default, I just know it's good practice to have it on action buttons such as submit, etc, that are supposed to trigger another set of actions (in our case it would be an SSE or hit an API endpoint). I've ran into a lot of issues with it double pressing as well when there wasn't a prevent default.